tox.ini

```diff
@@ -56,14 +56,22 @@ setenv =
     PIP_DISABLE_PIP_VERSION_CHECK=1
     VIRTUALENV_NO_DOWNLOAD=1
 
+    coverage: COVERAGE_FILE={toxworkdir}/coverage/coverage.{envname}
+    codecov: COVERAGE_FILE={toxworkdir}/coverage/coverage
```

Comment:
What going on here is we create a `coverage.<envname>` file for each environment.

When the `codecov` environment runs `coverage combine`, it'll combine any `coverage.*` files into a single `coverage` file.